.github/workflows/docs-ci.yaml

```diff
@@ -0,0 +1,53 @@
+name: documentation CI
+
+on:
+  workflow_call:
+    inputs:
+      git_tag:
+        type: string
+        description: The git tag (version) from the calling workflow
+        required: true
+
+jobs:
+  docs-ci:
+    runs-on: ubuntu-latest
+    env:
+      GH_TOKEN: ${{ secrets.GH_TOKEN_NVIDIA_CI_CD }}
+      TAG: ${{ inputs.git_tag || 'latest' }}
+      PR_TITLE_PREFIX: "task: update documentation for"
+    steps:
+    - uses: actions/checkout@v4
+      with:
+        token: ${{ secrets.GH_TOKEN_NVIDIA_CI_CD }}  # token must be explicitly set here for push to work in following step
+    - name: Make docs
```

Comment:
I think you have to set up the `TAG` environment var for it to work.

Reply:
done on line 16.

```yaml
      TAG: ${{ inputs.git_tag || 'latest' }}
```